c/include/libsbp/v4/sbp_msg.h

```diff
@@ -831,6 +831,8 @@ static inline s8 sbp_message_encode(uint8_t *buf, uint8_t len,
       return sbp_msg_vel_ned_encode(buf, len, n_written, &msg->vel_ned);
     case SbpMsgWheeltick:
       return sbp_msg_wheeltick_encode(buf, len, n_written, &msg->wheeltick);
+    case SbpMsgAll:
```

Comment:
See https://github.com/swift-nav/libsbp/blob/master/generator/sbpg/targets/resources/c/v4/sbp_msg_template.h#L63 for where these functions are generated